workspaces/core/src/banner/private/Multiline.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export interface Props {
+  description: string;
+  title?: string;
+  iconHeader?: React.ReactElement;
+  buttons?: React.ReactElement[];
+}
+
+const Multiline: React.FC<Props> = ({
+  description,
+  title,
+  iconHeader,
+  buttons
+}) => {
+  return (<>
+    {iconHeader && (<div className='w-full bg-goten flex rounded-lg justify-between'>
+      <div className='flex-col p-4'>
+        <div className='flex mb-2 justify-between'>
+          <div className='font-semibold'>{title}</div>
+        </div>
+        <div className='block align-middle items-center text-trunks'>{description}</div>
+      </div>
+      <div className='flex justify-end items-baseline p-4 cursor-pointer'>{iconHeader}</div>
+    </div>)}
+    {!iconHeader && buttons?.length && (<div className='w-full bg-goten sm:flex rounded-lg justify-between'>
+       <div className='flex-col p-6'>
+         <div className='flex mb-2 justify-between'>
+           <div className='font-semibold'>{title}</div>
+         </div>
+         <div className='block align-middle items-center text-trunks'>{description}</div>
```

Comment:
Please don't use inheritance for styling text. I mean put class for fontSize and lineHeight in element where it uses, not for the parent. 
And please use proper semantic html element for text

`<p className='text-moon-14 block align-middle items-center text-trunks'>{description}</p>`

Reply:
@yarema184  well here the case is that we don't control `description` and `title` elements as they are provided via props. the only way to ensure that these will be styles properly and aligned with the design is to use classes for styling text on wrapping parent elements. alternative is to force developers that use our component to style their title and description for themselves which is not ideal i guess... do you have any other suggestion?